src/ui/templates/insurance/account/password-reset/link-expired.njk

```diff
@@ -0,0 +1,23 @@
+{% extends 'index.njk' %}
+
+{% block pageTitle %}
```

Comment:
do we need a back button on these pages?

Reply:
No, if a link (from email) has expired or is invalid we redirect to this page. A back link would not have a HREF unless the page is refreshed. Have discussed with design